dictionary.txt

```diff
@@ -235,3 +235,123 @@ webkit-sticky
 webkit-text-fill-color
 whatsApp
 xmlns
+isHeight100Vh
+mobilePadding
+.swiper-slide-thumb-active
+ReactComponent
+RightArrorCircle
+StyledRightArrorCircle
+Swiper2
+.swiper-slide
+.swiper
+.mySwiper2
+.mySwiper
+Swiper
+FreeMode
+SwiperSlide
+thumbsSwiper
+setThumbsSwiper
+freeMode
+watchSlidesProgress
+PaginationBullet
+swiper-navigation-color
+onSwiper
+swiper-pagination-color
+Slider2
+ScrollContainer
+ZoomOut
+ZoomInScrollOut
+StickyIn
+FadeIn
+ZoomIn
+CustomZoom
+FadeUp
+lateralPadding
+ScrollPage
+fontSize
+FadeUpScrollOut
+MoveIn
+MoveOut
+br
+FadeAnimation
+ZoomAnimation
+transformOrigin
+user-scalable
+isTablet
+isLast
+useMediaQuery
+FidaLogo
+useMediaQuery
+ButtonWrapper
+maxWidth840
+maxWidth1140
+ViewportWrapper
+LightLinesBg
+FilledSquaresBg
+FirstLineWrapper
+eslint-disable
+linesBg
+LinesBg
```

Comment:
is it supposed to be both lower case and upper case?

Reply:
Yes. 'linesBg' is a svg file and 'LinesBg' is a styled React component.